docs/resources/security_monitoring_rule.md

```diff
@@ -177,6 +202,20 @@ Optional:
 - `default_rule_id` (String) Default Rule ID of the signal to correlate. This value is READ-ONLY.
 - `name` (String) Name of the query. Not compatible with `new_value` aggregations.
 
+
+<a id="nestedblock--third_party_case"></a>
+### Nested Schema for `third_party_case`
+
+Required:
+
+- `status` (String) Severity of the Security Signal. Valid values are `info`, `low`, `medium`, `high`, `critical`.
+
+Optional:
+
+- `name` (String) Name of the case.
+- `notifications` (List of String) Notification targets for each rule case.
+- `query` (String) A query to associate a third party event to this case.
```

Comment:
```suggestion
- `query` (String) A query to associate a third-party event to this case.
```

Reply:
Thanks! Applied in https://github.com/DataDog/terraform-provider-datadog/pull/2231/commits/1e4872ae66b9b61487da7cfbdc524101cda4f968.